packages/dts-gen/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@chevrotain/dts-gen",
+  "version": "8.0.1",
+  "description": "Generates type definitions for chevrotain CST nodes",
+  "keywords": [],
+  "bugs": {
+    "url": "https://github.com/Chevrotain/chevrotain/issues"
+  },
+  "license": "Apache-2.0",
+  "author": {
+    "name": "Shahar Soel"
```

Comment:
maybe we should ditch the author section here

Reply:
or maybe in the whole project (but not as part of this PR)

Reply:
kk